src/Loader.php

```diff
@@ -252,11 +248,9 @@ public static function load_features() {
 	 * @todo The entry point for the embed needs moved to this class as well.
 	 */
 	public static function register_page_handler() {
-		$id = self::is_feature_enabled( 'homescreen' ) ? 'woocommerce-home' : 'woocommerce-dashboard';
-
 		wc_admin_register_page(
 			array(
-				'id'         => $id, // Expected to be overridden if dashboard is enabled.
+				'id'         => 'homescreen',
```

Comment:
Should we keep this as `woocommerce-home` or does it need to be different than the page registration in `src/Features/AnalyticsDashboard.php`? Or maybe we can do away with one?

Reply:
Oddly enough, in the navigation project this entry is causing issues because the `title` property below is `null`. I'm wondering if its overwriting the other one.

Reply:
Everything seems to work fine if I remove this registration, done in 3a7e243